packages/shared-lib/src/util/wait.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * waits for the provided number of milliseconds
+ * before resolving the promise and allowing
+ * your code to continue.
+ * Spiritually similar to C#'s Thread.Sleep(amount)
+ */
+export function wait(amount: number) {
+  return new Promise<void>((resolve) => {
+    setTimeout(resolve, amount);
+  });
```

Comment:
I have a fun a deeply esoteric Node trick here that's a fun party trick, remind me at some point and I'll show you.